### PR TITLE
Update Transaction Preview Gas Used view - Add TP example to the Docs

### PR DESCRIPTION
**Pull request by Adamj1232**
### Description

To test:
- Pull down this branch
- `yarn && yarn dev`
- In a separate console `cd docs/` from the root of the project
- `yarn link @web3-onboard/transaction-preview && yarn && yarn dev`
- Navigate to `http://localhost:3000/docs/modules/transaction-preview#usage-with-web3-onboard-core-package` 
- Click Preview Transaction

This PR includes a couple bug fixes to the core package for backwards compatibility including:

- returning the `sidebar.heading` to the connect view
- An update to side `background-color` -> `background`

Updating the Transaction Preview gas calculation, view and functionality to get standalone working properly(required for docs usage outside of W3O integration)

Screen Shot 2023-02-15 at 10 15 53 AM

Adding a RP example button to the docs:

Screen Shot 2023-02-15 at 10 34 57 AM

### Checklist
- [x] The version field in `package.json` of the package you have made changes in is incremented following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [x] I have run `yarn file-check`, `yarn type-check` & `yarn build` to confirm there are not any associated errors
- [x] This PR passes the Circle CI checks

### If this PR includes changes to add an injected wallet or SDK wallet module: 
Please complete the following using the internal demo package.
To run this demo use the command `yarn && yarn dev` to get the project running at `http://localhost:8080/`

### CODE REVIEW

**Review by leightkt — APPROVED** [review]
I wasn't sure how to test the sidebar backward compatibility- but everything else is looking good.

**Comment by Adamj1232** [comment]
> I wasn't sure how to test the sidebar backward compatibility- but everything else is looking good.

Thanks @leightkt !
The `sidebar.heading` was removed in a previous PR so I added back for backwards compatibility. Sorry should have been more clear there. The little chuck of code controlling that area defaults to an empty string as does the map in the i18n

File: packages/core/src/views/connect/Sidebar.svelte [hunk 1]
```diff
@@ -150,6 +156,13 @@
           {@html defaultBnIcon}
         {/if}
       </div>
+      {#if $_(`connect.${step}.sidebar.heading`, { default: '' })}
+        <h2 class="heading">
+          {$_(`connect.${step}.sidebar.heading`, {
+            default: heading
+          })}
+        </h2>
```
Comment (gesquinca) [inline packages/core/src/views/connect/Sidebar.svelte:164]: We should change this element to a div. There's not really a reason for this to be an h2.

File: packages/transaction-preview/src/index.ts [hunk 2]
```diff
@@ -269,13 +269,48 @@ const mountTransactionPreview = (simResponse: MultiSimOutput) => {
...
+
+        /* SHADOWS */
+        --shadow-1: 0px 4px 12px rgba(0, 0, 0, 0.1);
+
+        /* MODAL POSITIONING */
+        --tp-z-index: 10;
+
+      }
+    </style>
+  `
```
Comment (gesquinca) [inline packages/transaction-preview/src/index.ts:302]: Any chance we can move this css to the style tag in packages/transaction-preview/src/views/Index.svelte ?

**Review by gesquinca — COMMENTED** [review]
Some small suggestions. @Adamj1232 let me know what you think!

File: packages/core/src/views/connect/Sidebar.svelte (hunk 1 above)
Comment (gesquinca) [reply packages/core/src/views/connect/Sidebar.svelte:164]: We could also update the class name to "sidebar-heading" to be more specific.

**Review by gesquinca — APPROVED** [review]

File: packages/transaction-preview/src/index.ts (hunk 2 above)
Comment (Adamj1232) [reply packages/transaction-preview/src/index.ts]: Good call, done!

**Pull request merged**